android/app/src/main/java/app/organicmaps/util/StringUtils.java

```diff
@@ -19,6 +19,11 @@ public static String formatUsingUsLocale(String pattern, Object... args)
     return String.format(Locale.US, pattern, args);
   }
```

Comment:
Do we still need formatUsingUsLocale? If yes, where is it used?

Reply:
https://github.com/organicmaps/organicmaps/blob/db7bc664c53f0d4c77c74ec2886849e1d56e812c/android/app/src/main/java/app/organicmaps/downloader/ChunkTask.java#L151

Reply:
Hi @vng, sorry for not having answered you sooner...

> Do we still need formatUsingUsLocale? If yes, where is it used?

The only remaining place where the `StringUtils.formatUsingUsLocale()` function is used to format a float value is here...

https://github.com/organicmaps/organicmaps/blob/bc62dafa59649f64c83bff4b6743866ab0a62fd1/android/app/src/main/java/app/organicmaps/downloader/CountryItem.java#L170

... which is used to convert to string the 'CountryItem' java class.